script/semantic-release-npm-prepare.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable no-console */
+
+/**
+ * Approximating functionality from:
+ * https://github.com/semantic-release/npm/blob/master/lib/prepare.js#L14
+ *
+ * This script will update the package.json version to the version.
+ * To be used with @semantic-release/exec before @semantic-release/git
+ */
+
+const fs = require('fs');
+
+// args, get version from semantic-release
+const version = process.argv[2];
+
+// load package.json
+console.log('Updating package.json version to: ', version);
+const packageJson = JSON.parse(fs.readFileSync('./package.json', 'utf8'));
+packageJson.version = version;
+
+console.log('Writing new package.json contents.');
```

Comment:
is this dupe work of `make update_npmversion` (makefile ln 58)

Reply:
Not exactly.. this script doesn't update the root pacjage.json version which `@semantic-release/npm` would. I moved the functionality into a bash script inside the /scripts folder. The script updates both root and /website package.json files when ran.